index.js

```diff
@@ -0,0 +1,46 @@
+const bodyParser = require('body-parser')
+const express = require('express')
+const Movies = require('./models')
```

Comment:
You would need to name this models since below you reference `models.Movies` right now there is nothing called `models` so you would get an error `cannot find property Movies of undefined`